src/tests.rs

```diff
@@ -1,7 +1,8 @@
 use actix_web::{dev::Service, http::StatusCode, test, web, App, HttpResponse, Responder};
 
-#[test]
-fn builder_test() {
+// This won't compile without `async` for some reason
```

Comment:
Interesting... this should not be async and I'm against changing this test unless there's a good reason.

Reply:
I have no clue why it happens, but it wasn't compiling for me on 1.56.1, I'll do some research though.

Reply:
Hm it seems this was an error on IntelliJ's part. The solution is to move the `use actix_web::test` into the function body. Something fishy is going on there.

I've included that in the commit so future users will not run into the issue